.github/workflows/update_eu_policies.yml

```diff
@@ -0,0 +1,31 @@
+name: Create EU_Policies promotion pull request
+on:
+  push:
+    branches:
+      - main
+jobs:
+  EU_PoliciesPromotion:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: EU_Policies
+      - name: Merge in master
+        run: |
+          git config --global user.email "support@flexera.com"
+          git config --global user.name "Flexera Support"
+          git fetch origin master:master
+          git merge master --strategy-option theirs --allow-unrelated-histories
+      - name: CreatePullRequest
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
```

Comment:
I think you should set the `title` for this to something like "Update EU_Policies with latest changes from master"

Reply:
Looks like you could also set `body` with some more relevant information as well.